### ISSUE TITLE
Add assign score component

### ISSUE THREAD

**Issue by ERnsTL** [ERnsTL/flowd#294]
* assign an incoming data value a score
* use cases:
  * red/yellow/green decision for eg. monitoring system
  * assign score based on input value interval
  * assign score based on formula (that might get complicated), eg. for every 100 input units, give 1 score and subtract 2 so that the score is given only above 200 input units.
* general way to solve this:  apply a lambda function to every packet - but which language? wasm?
* input rules like:
  * value min, max
  * contains
  * not contains
  * and and or rules
* basically a configurable if statement which assigns a score

**Comment by ERnsTL** [comment]
Solved using template component #102
